matrix-guide.md

```diff
@@ -122,3 +122,18 @@ Suggested settings to change:
 [#tc39-beginners]: https://matrix.to/#/#tc39-beginners:matrix.org
 [#tc39-inclusion]: https://matrix.to/#/#tc39-inclusion:matrix.org
 [#tc39-website]: https://matrix.to/#/#tc39-website:matrix.org
+
+# Creating a new channel on Matrix for TC39 use
+(This assumes that you're using the Element client)
+
+- Click the plus icon next to "Rooms" in the sidebar
+- Give the room a name, like "TC39 Delegates"
+- Give the room a topic, if desired
+- Enable "Make this room public"
+- Make the room address a lowercase, dashed version of the name you input earlier, like "tc39-delegates"
+- Click "Create Room"
+- Click the three-dot menu button that appears when you hover over the new room in the sidebar, then click "Settings" in the menu that appears
+- In General, add the TC39 logo as the room's image. You can find the logo [here](https://avatars0.githubusercontent.com/u/1725583?s=280&v=4).
+- At the bottom of General, enable URL previews by default for participants in this room
+- In Security & Privacy, set "Who can access this room?" to "Anyone who knows the room's link, including guests". Also set "Who can read history?" to "Anyone".
+- In Advanced, copy the internal room ID. Go back to General, and add a link to the logs in the room topic, like so: "Public logs at https://view.matrix.org/room/!WgJwmjBNZEXhJnXHXw:matrix.org/".
```

Comment:
maybe add something about asking the chairs to add the channel to the TC39 space?